### PR TITLE
docs: rename uat to preprod

### PR DESCRIPTION
**Pull request by stephanemillaire-ssc**
- Renamed uat to preprod in required documents and images.

### CODE REVIEW

**Review by lucstjean-ssc — DISMISSED** [review]
/lgtm

File: Developer/Guide.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ Example of policy use cases
 
 # Shared Services Canada - GCP Landing zone
 
-SSC has built 4 distinct GCP organizations to isolate each environment from each other. The environments are experimentation, DEV, UAT and PROD.
+SSC has built 4 distinct GCP organizations to isolate each environment from each other. The environments are experimentation, DEV, PREPROD and PROD.
 TODO: update diagram to reflect experimentation
```
Comment (alaincormier-ssc) [inline Developer/Guide.md:46]: looks like the diagram is updated now, the "TODO..." could be removed

**Review by ovidiuasiminei-ssc — APPROVED** [review]
lgtm

File: Developer/Guide.md (hunk 1 above)
Comment (stephanemillaire-ssc) [reply Developer/Guide.md]: removed identified TODO

**Review by alaincormier-ssc — APPROVED** [review]
nice work!

**Review by davelanglois-ssc — APPROVED** [review]

**Comment by davelanglois-ssc** [comment]
superbe !

**Pull request merged**